### DISCUSSION TITLE
Feature Request: Button for changing OBS Studio scenes

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#391] [Ideas] [answered]
Hello brother, I would like to suggest a change in the way we change the SCENES we create in OBS Studio. With the use of some of the SCENES created in OBS Studio (Stage, Speaker, Reader, Table), the SCENES in the current format end up taking a while to change. I believe that if they were a button or something similar, it would make it easier to change scenes when changing participants. See a suggestion at the bottom of the image. And each [Scene] button will have a caption, and will have a "bright" color when active.

![image](https://user-images.githubusercontent.com/90984283/186304935-86e13aec-9d54-4b57-b203-f1507f6ebd67.png)

Aqui uso assim as CENAS feitas pelo OBS Studio.
[Here I use the SCENES made by OBS Studio like this.]
![image](https://user-images.githubusercontent.com/90984283/174387310-1d9d4c46-2095-433f-8f54-de5846aa1ce5.png)

**Comment by mtdvlpr** [comment, marked as answer]
I like it! And to make sure it fits, you can set a max and if there are more scenes than the max you can display it as a dropdown (current behavior)

**Reply by e-Feitosa** [reply]
Gostei da sua sugestão de colocar visível tipo 5 botões com as cenas definidas e as demais cenas igual ao que já temos [drop-down]

[I like your suggestion of putting visible, like 5 buttons with the set scenes and the other scenes like we already have [drop-down].]

**Reply by e-Feitosa** [reply]
@sircharlo What do you think of this update? Is it possible?

**Reply by sircharlo** [reply]
For sure it's possible! 

As to when I will have time to add it, that's a different matter 😆 

But I do like the idea very much.

**Reply by mubarakmachado** [reply]
I really like this idea too.

**Reply by e-Feitosa** [reply]
Most importantly it is working well, these details will improve the program even more, be happy that it will be on the list for future updates. I am very grateful for your valuable time on this project.

**Comment by e-Feitosa** [comment]
It was excellent!

![image](https://user-images.githubusercontent.com/90984283/186304818-203ec2b8-0086-460e-a343-7bcdd251b7d2.png)
⬆️
